build_tools/packaging/linux/package.json

```diff
@@ -207,103 +256,2443 @@
+      "libc6",
+      "rocm-blas"
+    ],
+    "RPMRequires": [
+      "python3",
+      "glibc",
+      "rocm-blas"
+    ],
+    "Maintainer": "ROCm Dev Support <rocm-dev.support@amd.com>",
+    "Description_Short": "AMD library for BLAS, sparse and solver in ROCm",
+    "Description_Long": "blas, sparse and solver development files",
+    "Section": "devel",
+    "Priority": "optional",
+    "Group": "unknown",
+    "License": "MIT",
+    "Vendor": "Advanced Micro Devices, Inc",
+    "Homepage": "https://github.com/ROCm/rocm-libraries",
+    "Components": [
+      "dev"
+    ],
+    "Includes": [
+      "rocsparse-devel",
+      "hipsparse-devel",
+      "hipsolver-devel",
+      "rocsolver-devel",
+      "hipblas-devel",
+      "rocblas-devel",
+      "hipblaslt-devel"
+    ],
+    "Artifact": "blas",
+    "Gfxarch": "True",
+    "Composite": "Yes"
+  },
+  {
+    "Package": "rocm-fft",
+    "Version": "",
+    "Architecture": "amd64",
+    "BuildArch": "x86_64",
+    "DEBDepends": [
+      "python3",
+      "libc6"
+    ],
+    "RPMRequires": [
+      "python3",
+      "glibc"
+    ],
+    "Maintainer": "ROCm Dev Support <rocm-dev.support@amd.com>",
+    "Description_Short": "ROCm FFT library",
+    "Description_Long": "rocfft and hipfft library files",
+    "Section": "devel",
+    "Priority": "optional",
+    "Group": "unknown",
+    "License": "MIT",
+    "Vendor": "Advanced Micro Devices, Inc",
+    "Homepage": "https://github.com/ROCm/rocm-libraries",
+    "Components": [
+      "lib",
+      "run",
+      "doc"
+    ],
+    "Includes": [
+      "rocfft",
+      "hipfft"
+    ],
+    "Artifact": "base",
+    "Gfxarch": "True",
+    "Composite": "Yes"
+  },
+  {
+    "Package": "rocm-fft-devel",
+    "Version": "",
+    "Architecture": "amd64",
+    "BuildArch": "x86_64",
+    "DEBDepends": [
+      "python3",
+      "libc6",
+      "rocm-fft"
+    ],
+    "RPMRequires": [
+      "python3",
+      "glibc",
+      "rocm-fft"
+    ],
+    "Maintainer": "ROCm Dev Support <rocm-dev.support@amd.com>",
+    "Description_Short": "ROCm FFT library",
+    "Description_Long": "rocfft and hipfft developement header files",
+    "Section": "devel",
+    "Priority": "optional",
+    "Group": "unknown",
+    "License": "MIT",
+    "Vendor": "Advanced Micro Devices, Inc",
+    "Homepage": "https://github.com/ROCm/rocm-libraries",
+    "Components": [
+      "dev"
+    ],
+    "Includes": [
+      "rocfft-devel",
+      "hipfft-devel"
+    ],
+    "Artifact": "fft",
+    "Gfxarch": "True",
+    "Composite": "Yes"
+  },
+
+  {
+    "Package": "rocm-rand",
+    "Version": "",
+    "Architecture": "amd64",
+    "BuildArch": "x86_64",
+    "DEBDepends": [
+      "python3",
+      "libc6"
+    ],
+    "RPMRequires": [
+      "python3",
+      "glibc"
+    ],
+    "Maintainer": "ROCm Dev Support <rocm-dev.support@amd.com>",
+    "Description_Short": " ROCm random number generation library",
+    "Description_Long": "Contains rocrand and hiprand libraries",
```

Comment:
leading space in the short description? Here and in a few other places

Reply:
done